tutorial.md

```diff
@@ -0,0 +1,45 @@
+# Charmed MongoDB tutorial:
```

Comment:
note to self: might be good to have output of the commands so user can check their progress along the way